nautobot/docs/development/core/kubernetes-job-support.md

```diff
@@ -0,0 +1,509 @@
+# Kubernetes Job Support
+
+Kubernetes Job Support is added in Nautobot v2.4.0. This documentation is a end-to-end development guide on topics from how to set up your local kubernetes cluster with [minikube](https://minikube.sigs.k8s.io/docs/) and how to run a nautobot job in a remote kubernetes job pod.
+
+## Preliminary Setup
+
+First we need to install minikube, go to the offical [get started page](https://minikube.sigs.k8s.io/docs/start/?arch=%2Fmacos%2Farm64%2Fstable%2Fbinary+download) to learn how to download minikube.
```

Comment:
Looks like that URL is specific to mac/arm64 users - maybe replace with something more generic?